src/main/java/ch/njol/skript/expressions/ExprMendingRepairAmount.java

```diff
@@ -0,0 +1,116 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerItemMendEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Mending Repair Amount")
+@Description({"The number of durability points an item is to be repaired in a mending event.",
+			" Modifying the repair amount will affect how much experience is given to the player after mending."})
+@Examples({"on item mend:",
+		"\tset the mending repair amount to 100"})
+@Since("INSERT VERSION")
+public class ExprMendingRepairAmount extends SimpleExpression<Number> {
+
+	static {
+		Skript.registerExpression(ExprMendingRepairAmount.class, Number.class, ExpressionType.SIMPLE, "[the] [mending] repair amount");
+	}
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		if (!ScriptLoader.isCurrentEvent(PlayerItemMendEvent.class)) {
+			Skript.error("The 'mending repair amount' is only usable in item mend events", ErrorQuality.SEMANTIC_ERROR);
+			return false;
+		}
+		return true;
+	}
+
+	@Override
+	protected Number[] get(final Event e) {
+		return new Number[]{((PlayerItemMendEvent) e).getRepairAmount()};
+	}
+
+	@Nullable
+	@Override
+	public Class<?>[] acceptChange(ChangeMode mode) {
+		switch (mode) {
```

Comment:
Might the RESET ChangeMode be applicable here?

Reply:
It might be possible if I get the ExperienceOrb actually. I'll have to test that out.